e2e/tests/008.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set the new value for maxUplinkThroughput as a parameter
+
+set -o pipefail
+set -o errexit
+set -o nounset
+[[ "${DEBUG:-false}" != "true" ]] || set -o xtrace
+
+new_capacity_value=$1
+
+export HOME=${HOME:-/home/ubuntu/}
+export E2EDIR=${E2EDIR:-$HOME/test-infra/e2e}
+export TESTDIR=${TESTDIR:-$E2EDIR/tests}
+export LIBDIR=${LIBDIR:-$E2EDIR/lib}
+
+source "${LIBDIR}/k8s.sh"
+
+kubeconfig="$HOME/.kube/config"
+
+#upf_deployment_pkg=$(kpt alpha rpkg get --name free5gc-packages  --revision v3 -o jsonpath='{.metadata.name}')
+
+#upf_pkg_rev=$(kpt alpha rpkg clone -n default "$upf_deployment_pkg" --repository mgmt edge01 | cut -f 3 -d ' ')
+
+upf_deployment_pkg=$(kpt alpha rpkg get --name free5gc-upf | grep edge01 | grep latest | cut -d ' ' -f 1)
+
+upf_pkg_rev=$(kpt alpha rpkg copy $upf_pkg_rev  --repository edge01 | cut -f 3 -d ' ')
+
+kpt alpha rpkg pull -n default "$upf_pkg_rev" edge01_upf_scaling
+
+kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.2.0 edge01_upf_scaling -- by-path='spec.maxUplinkThroughput' by-value=10G put-value=$new_capacity_value
+kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.2.0 edge01_upf_scaling -- by-path='spec.maxDownlinkThroughput' by-value=10G put-value=$new_capacity_value
+
+kpt alpha rpkg push -n default "$upf_pkg_rev" edge01
+
+kpt alpha rpkg propose -n default "$upf_pkg_rev"
+
+kpt alpha rpkg approve -n default "$upf_pkg_rev"
+
+kubectl get secret edge01-kubeconfig -o jsonpath='{.data.value}' | base64 -d > edge01-kubeconfig
```

Comment:
I added a k8s_get_capi_kubeconfig function to make this easier, though this is fine too